lib/ts/recipe/passwordless/types.ts

```diff
@@ -263,6 +263,7 @@ export type UserInput = (
           validatePhoneNumber?: (phoneNumber: string) => Promise<string | undefined> | string | undefined;
 
           signInUpFeature?: SignInUpFeatureConfigInput & {
+              defaultCountry?: CountryCode;
```

Comment:
Is this a bug fix for passwordless recipe? This needs to go into another PR which we can release separately.